.travis.yml

```diff
@@ -41,8 +41,10 @@ x-pyenv-env: &x-pyenv-env >
   PATH="${PYENV_ROOT}/shims:${PATH}"
 
 x-pyenv-install: &x-pyenv-install |
-  # Ensure we have the targeted Python for tox to find on $PATH
-  PYENV="${PYENV_ROOT}/bin/pyenv"
+  # Ensure we have the targeted Python for tox to find on $PATH.
+  # Relative symlinks don't work on Git Bash, which is why
```

Comment:
What is "Git Bash"?

Reply:
[Git Bash](https://gitforwindows.org/) is the [default shell that the Travis Windows image uses](https://docs.travis-ci.com/user/reference/windows/#git-bash)

Reply:
Hmm, wow. Ok. Thanks - TIL.

I mean, when I left Windows dev all that time ago cygwin worked just fine as a package manager / environment and had real git and real bash.